roles/openshift_node_upgrade/README.md

```diff
@@ -0,0 +1,108 @@
+| l_docker_upgrade               |                       |                                                        |
+| node_config_hook               |                       |                                                        |
+| openshift.docker.gte_1_10      |                       |                                                        |
+| openshift_image_tag            |                       | Set by openshift_version role                          |
+| openshift_pkg_version          |                       | Set by openshift_version role                          |
+| openshift_release              |                       | Set by openshift_version role                          |
+| skip_docker_restart            |                       |                                                        |
+| openshift_cloudprovider_kind   |                       |                                                        |
+
+From openshift.common:
+
+| Name                               |  Default Value      |                     |
+|------------------------------------|---------------------|---------------------|
+| openshift.common.config_base       |---------------------|---------------------|
+| openshift.common.data_dir          |---------------------|---------------------|
+| openshift.common.hostname          |---------------------|---------------------|
+| openshift.common.http_proxy        |---------------------|---------------------|
+| openshift.common.is_atomic         |---------------------|---------------------|
+| openshift.common.is_containerized  |---------------------|---------------------|
+| openshift.common.portal_net        |---------------------|---------------------|
+| openshift.common.service_type      |---------------------|---------------------|
+| openshift.common.use_openshift_sdn |---------------------|---------------------|
+
+From openshift.master:
+
+| Name                               |  Default Value      |                     |
+|------------------------------------|---------------------|---------------------|
+| openshift.master.api_port          |---------------------|---------------------|
+
+From openshift.node:
+
+| Name                               |  Default Value      |                     |
+|------------------------------------|---------------------|---------------------|
+| openshift.node.debug_level         |---------------------|---------------------|
+| openshift.node.node_image          |---------------------|---------------------|
+| openshift.node.ovs_image           |---------------------|---------------------|
+
+
+Dependencies
+------------
+openshift_common
+
+TODO
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+```
+---
+- name: Upgrade nodes
+  hosts: oo_nodes_to_upgrade
```

Comment:
At some point how this group is calculated should be documented here in the readme, specifically how you can run node upgrade on only nodes with a specific label, and with a value for serial.